R/geom-bin2d.r

```diff
@@ -41,3 +41,33 @@ geom_bin2d <- function(mapping = NULL, data = NULL,
     )
   )
 }
+
+#' @rdname animint2-gganimintproto
+#' @format NULL
+#' @usage NULL
+#' @export
+#' @include geom-rect.r
+GeomBind2d <- gganimintproto("GeomBind2d", GeomRect, 
```

Comment:
why do you need to add this?

Reply:
I added this because bin2d is not supported in animint2. So, there might be chances that we'll be adding support for this. So, there should be a class where the operations can be done. And I had to show this message to users when $pre_process is being invoked.
```r
stop("bin2d is not supported in animint. Try using geom_tile() and binning the data yourself.")
```